### ISSUE TITLE
language selector doesn't appear on custom post types created using Advanced Custom Fields plugin

### ISSUE THREAD

**Issue by catharsis71** [Swiss-Mac-User/language-locale-overwrite#3]
I am helping someone with their mixed-language Wordpress and this plugin seems like a good start but I'm only able to change the language on normal Posts and Pages.  They are using custom post types via ACF and the language selector does not show up

for example I can edit the language on the [category page](https://ether.lricountry.com/proto-galardica/) but not on the [sub page](https://ether.lricountry.com/proto-galardica/cantautores-repentinos/)

could this support potentially be added?

**Comment by Swiss-Mac-User** [comment]
Hi @catharsis71 

That's true, it currently specifically uses only WP default Post and Page types. The reason is, that I don't have custom types, so I was not able to test the behaviour reliably.

What you can do to try to get it work:

* Open the plugin's file "`language-locale-overwrite.php`" in a TextEditor
* Find line :133 that starts with `add_meta_box(`
* Remove the following part from the very end of this line: `, ['post', 'page']` (make sure to include the `,` comma!). Make the linke look basically as follows:<br>`add_meta_box( 'language-locale-overwrite', __( 'Language Locale', 'language_locale_overwrite' ), 'show_language_locale_meta_box' );`
* Save the file
* In WordPress, check if not the Language Locale fields is available on the custom post types

<img width="1378" alt="add-language-locale-to-custom-post-types" src="https://github.com/Swiss-Mac-User/language-locale-overwrite/assets/47624464/95ed78b3-b758-4204-bea7-997c72112776">
-
 
**Let me know if that worked**, I may incorporate this change then generally to the plugin's code in a next update.

**Comment by Swiss-Mac-User** [comment]
I looked into it briefly, but it's difficult to replicate: even with a Custom Post Type plugin, I didn't manage to get a custom post type to show up / be usable at all (in WP Admin).

In addition, I identified a function in the plugin's code that may also need to be extended for proper «Custom Post Types»-support: https://github.com/Swiss-Mac-User/language-locale-overwrite/blob/a578bc004f16eeb4b646620e99573fe93bc13c6d/language-locale-overwrite.php#L209

Long story short: **Would appreciate some help on this** from someone with a theme that has and supports custom post types.

**Comment by Swiss-Mac-User** [comment]
The [latest Release 2.4.0](https://github.com/Swiss-Mac-User/language-locale-overwrite/releases/tag/2.4.0) allows to use the locale fields now also **on custom Post- and Page types** in WordPress.
